coderadar-test/src/test/java/io/reflectoring/coderadar/rest/useradministration/teams/ListProjectsForTeamControllerIntegrationTest.java

```diff
@@ -0,0 +1,62 @@
+package io.reflectoring.coderadar.rest.useradministration.teams;
+
+import static io.reflectoring.coderadar.rest.JsonHelper.fromJson;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+
+import io.reflectoring.coderadar.graph.projectadministration.domain.ProjectEntity;
+import io.reflectoring.coderadar.graph.projectadministration.project.repository.ProjectRepository;
+import io.reflectoring.coderadar.graph.useradministration.domain.TeamEntity;
+import io.reflectoring.coderadar.graph.useradministration.repository.TeamRepository;
+import io.reflectoring.coderadar.rest.ControllerTestTemplate;
+import io.reflectoring.coderadar.rest.domain.GetProjectResponse;
+import java.util.Collections;
+import java.util.Date;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+class ListProjectsForTeamControllerIntegrationTest extends ControllerTestTemplate {
+
+  @Autowired private TeamRepository teamRepository;
+
+  @Autowired private ProjectRepository projectRepository;
+
+  private TeamEntity teamEntity;
+  private ProjectEntity testProject;
+
+  @BeforeEach
+  void setUp() {
+    projectRepository.deleteAll();
```

Comment:
Zu Beginn eines jeden Tests sollte die Datenbank leer sein, Aufräumarbeiten sollten entsprechend in der `@AfterEach`-Methode geschehen.

Reply:
In IntegrationTestTemplate ausgelagert.